sql/moz-fx-data-shared-prod/accounts_backend_external/nonprod_accounts_v1/schema.yaml

```diff
@@ -1,37 +1,47 @@
 fields:
-- name: uid
-  type: BYTES
+- description: |-
+    Account ID in hexadecimal format.
+    FxA stores this as bytes, for purposes of logging or integration
+    with other systems we convert it to a hex string.
+  name: uid
+  type: STRING
   mode: NULLABLE
```

Comment:
IMO it'd be more readable to put the description last rather than first.

Reply:
We should have a formatter for schema.yaml.